resteasy-jaxrs/src/main/java/org/jboss/resteasy/tracing/RESTEasyTracingLoggerImpl.java

```diff
@@ -1,13 +1,7 @@
 package org.jboss.resteasy.tracing;
 
 import org.jboss.logging.Logger;
-import org.jboss.resteasy.tracing.api.RESTEasyMsgTraceEvent;
-import org.jboss.resteasy.tracing.api.RESTEasyServerTracingEvent;
-import org.jboss.resteasy.tracing.api.RESTEasyTracing;
-import org.jboss.resteasy.tracing.api.RESTEasyTracingEvent;
-import org.jboss.resteasy.tracing.api.RESTEasyTracingInfo;
-import org.jboss.resteasy.tracing.api.RESTEasyTracingLevel;
-import org.jboss.resteasy.tracing.api.RESTEasyTracingMessage;
+import org.jboss.resteasy.tracing.api.*;
```

Comment:
Can you please avoid using * imports? It's more clear to know what exactly is used, especially given the stuff here comes from an optional dependency. Moreover, a checkstyle will soon be used and it will flag * imports.